pkgs/moduleit/module-definition.nix

```
@@ -567,14 +567,18 @@ in
 
     replit.buildModule =
       let
-
+        envWithMergedPath = (env: path:
+          let packagesPath = lib.makeBinPath config.packages;
+          in
+          if builtins.hasAttr "PATH" env
+          then env // { PATH = "${env.PATH}\:${packagesPath}"; }
```

```suggestion
          then env // { PATH = "${env.PATH}:${packagesPath}"; }
```
Is there a reason to escape the colon? I tried it either way and it seems the same...

---

Weird, it was not working for me with it. Let me try again.

---

lol, fine to keep it then, it doesn't break anything.